usr/share/rear/verify/VEEAM/default/400_verify_veeam.sh

```diff
@@ -0,0 +1,16 @@
+# 400_verify_veeam.sh
+# check that veeam vbrserver is actually available (ping)
+test "${VBRSRV}" || Error "Define VBRSRV (hostname or IP address) in $CONFIG_DIR/local.conf file"
+
+if test "$PING" ; then
+        if ping -c 1 "${VBRSRV}" >/dev/null 2>&1 ; then
+           Log "Veeam VBR Server: ${VBRSRV} seems to be up and running."
+        else
+           Error "Unable to contact Veeam VBR Server: ${VBRSRV}"
+        fi
+else
+        Log "Skipping ping test"
```

Comment:
This Log message alone is meaning less.
So better provide some context to make it meaningful like
```
Log "Skipping ping test for Veeam VBR Server $VBRSRV"
```